packages/nc-gui/components/cell/attachment/index.vue

```diff
@@ -209,15 +212,29 @@ const rowHeight = inject(RowHeightInj, ref(1.8))
               <div class="text-center w-full">{{ item.title }}</div>
             </template>
             <div v-if="isImage(item.title, item.mimetype ?? item.type)">
-              <div class="nc-attachment flex items-center justify-center" :class="{'ml-2':active}" @click.stop="selectedImage = item">
+              <div
+                class="nc-attachment flex items-center justify-center"
+                :class="{ 'ml-2': active }"
+                @click.stop="selectedImage = item"
+              >
                 <LazyCellAttachmentImage
-                  class="max-h-[1.8rem] max-w-[1.8rem]"
+                  class="h-[1.8rem] w-[1.8rem]"
                   :alt="item.title || `#${i}`"
+                  :class="{
+                    'h-11.5 w-[3.8rem]': rowHeight === 2,
+                    'h-[4.8rem] w-[5.8rem]': rowHeight === 4,
```

Comment:
@sreehari2003 If exact value is not required for all other h-[*] and w-[*] class, replace the with standard tailwind classes